lib/SystemStatistics.php

```diff
@@ -24,25 +24,38 @@
 
 use OC\Files\View;
 use OCP\IConfig;
+use OCP\App\IAppManager;
+use OC\App\AppStore\Fetcher\AppFetcher;
 
 class SystemStatistics {
 
 	/** @var IConfig */
 	private $config;
-
 	/** @var View view on data/ */
 	private $view;
+	/** @var appFetcher */
+	private $appFetcher;
+	/** @var appManager */
+	private $appManager;
 
 	/**
 	 * SystemStatistics constructor.
 	 *
-	 * @param IConfig $config
+ 	 * @param IConfig $config
+	 * @param IAppManager $appManager
 	 */
```

Comment:
remove the doc block, just duplicating the method signature anyway.
Or you add the `$appFetcher`